client/package.json

```diff
@@ -11,6 +11,7 @@
     "react-bootstrap": "^1.0.0-beta.14",
     "react-dom": "^16.9.0",
     "react-helmet": "^5.2.1",
+    "react-places-autocomplete": "^7.2.1",
```

Comment:
This is a nifty package that bundles most of the functionality we need to integrate with the Google Maps Places API in one component.